cf-oscore/src/main/java/org/eclipse/californium/oscore/Encryptor.java

```diff
@@ -186,9 +199,15 @@ protected static Encrypt0Message prepareCOSEStructure(byte[] confidential) {
 		int firstByte = 0x00;
 		ByteArrayOutputStream bRes = new ByteArrayOutputStream();
 
+		//If the Context ID should be included for this context, set its bit
+		if(ctx.getIncludeContextId() == true) {
```

Comment:
Please insert a space (' ') between "if" and "(".

Reply:
Fixed in commit c2bffa2b0b1144b65108adbd7f1cc08638928d44